boards.txt

```diff
@@ -632,46 +632,23 @@ feathers2.menu.CPUFreq.10.build.f_cpu=10000000L
 feathers2.menu.FlashMode.qio=QIO
 feathers2.menu.FlashMode.qio.build.flash_mode=dio
 feathers2.menu.FlashMode.qio.build.boot=qio
-feathers2.menu.FlashMode.dio=DIO
-feathers2.menu.FlashMode.dio.build.flash_mode=dio
-feathers2.menu.FlashMode.dio.build.boot=dio
-feathers2.menu.FlashMode.qout=QOUT
-feathers2.menu.FlashMode.qout.build.flash_mode=dout
-feathers2.menu.FlashMode.qout.build.boot=qout
-feathers2.menu.FlashMode.dout=DOUT
-feathers2.menu.FlashMode.dout.build.flash_mode=dout
-feathers2.menu.FlashMode.dout.build.boot=dout
 
 feathers2.menu.FlashFreq.80=80MHz
 feathers2.menu.FlashFreq.80.build.flash_freq=80m
-feathers2.menu.FlashFreq.40=40MHz
-feathers2.menu.FlashFreq.40.build.flash_freq=40m
 
 feathers2.menu.FlashSize.16M=16MB (128Mb)
 feathers2.menu.FlashSize.16M.build.flash_size=16MB
-feathers2.menu.FlashSize.4M=4MB (32Mb)
-feathers2.menu.FlashSize.4M.build.flash_size=4MB
 feathers2.menu.FlashSize.8M=8MB (64Mb)
 feathers2.menu.FlashSize.8M.build.flash_size=8MB
 feathers2.menu.FlashSize.8M.build.partitions=default_8MB
-feathers2.menu.FlashSize.2M=2MB (16Mb)
-feathers2.menu.FlashSize.2M.build.flash_size=2MB
-feathers2.menu.FlashSize.2M.build.partitions=minimal
 
 feathers2.menu.UploadSpeed.921600=921600
 feathers2.menu.UploadSpeed.921600.upload.speed=921600
 feathers2.menu.UploadSpeed.115200=115200
 feathers2.menu.UploadSpeed.115200.upload.speed=115200
-feathers2.menu.UploadSpeed.256000.windows=256000
-feathers2.menu.UploadSpeed.256000.upload.speed=256000
-feathers2.menu.UploadSpeed.230400.windows.upload.speed=256000
-feathers2.menu.UploadSpeed.230400=230400
-feathers2.menu.UploadSpeed.230400.upload.speed=230400
 feathers2.menu.UploadSpeed.460800.linux=460800
 feathers2.menu.UploadSpeed.460800.macosx=460800
-feathers2.menu.UploadSpeed.460800.upload.speed=460800
-feathers2.menu.UploadSpeed.512000.windows=512000
-feathers2.menu.UploadSpeed.512000.upload.speed=512000
+feathers2.menu.UploadSpeed.460800.upload.speed=46080
```

Comment:
I think you lost a zero at the end...

Reply:
eagle eye

Reply:
Thanks, I fixed that... but  broke everything on a rebase - I HATE GIT!